src/WebJobs.Script.WebHost/web.config

```diff
@@ -1,22 +1,27 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-
   <!-- To customize the asp.net core module uncomment and edit the following section. 
   For more info see https://go.microsoft.com/fwlink/?linkid=838655 -->
   <system.webServer>
     <handlers>
-      <remove name="aspNetCore"/>
-      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified"/>
+      <remove name="aspNetCore" />
```

Comment:
These are local changes that need not be merged. correct?

Reply:
How did those get in here again. *scrubs them out*